MediaBrowser.Model/Configuration/ServerConfiguration.cs

```diff
@@ -15,41 +15,184 @@ public class ServerConfiguration : BaseApplicationConfiguration
     {
         public const int DefaultHttpPort = 8096;
         public const int DefaultHttpsPort = 8920;
```

Comment:
Since these have documentation comments within `NetworkConfiguration` should we ad the same doc comments here too or skip them :thinking: 

/note: I am aware that this class was not a focus of this PR, just wanted to note it